Sources/SmileID/Classes/Networking/Models/PrepUpload.swift

```diff
@@ -4,38 +4,37 @@ public struct PrepUploadRequest: Codable {
     public var partnerParams: PartnerParams
     // Callback URL *must* be defined either within your Partner Portal or here
     public var callbackUrl: String? = SmileID.callbackUrl
+    // TODO - Michael will change this to a boolean
+    public var allowNewEnroll: String = "false"
     public var partnerId = SmileID.config.partnerId
     public var sourceSdk = "ios"
     public var sourceSdkVersion = SmileID.version
     public var timestamp = String(Date().millisecondsSince1970)
     public var signature = ""
-    /// backend is broken needs these as strings
-    /// I've also made this false until we have this properly
-    /// documented and done on both android and iOS
-    public var allowNewEnroll = "false"
     public var useEnrolledImage = false
     public var retry = "false" /// backend is broken needs these as strings
 
     public init(
         partnerParams: PartnerParams,
-        callbackUrl: String? = "",
+        callbackUrl: String? = SmileID.callbackUrl,
```

Comment:
I know this also needs to be fixed but do you want this change in this PR or a separate one?

Reply:
Here is fine. I will make sure I separate PRs in the future